cards/multilang-faq-accordion/component.js

```diff
@@ -68,11 +69,28 @@ class multilang_faq_accordionCardComponent extends BaseCard['multilang-faq-accor
 
     const contentEl = this._container.querySelector(accordionContentSelector);
     let isExpanded = this._container.querySelector(`.${accordionExpandedClass}`);
-    contentEl.style.height = `${isExpanded ? contentEl.scrollHeight : 0}px`;
+    let stayExpanded = this._container.querySelector(`.${stayExpandedClass}`);
+
+    const cardEl = this._container.querySelector(accordionCardSelector);
     const linkEls = contentEl.querySelectorAll('a');
+
+    if (stayExpanded && this._state._state.feedbackSubmitted) {
```

Comment:
I know what this._state is, but what is this._state._state?

Reply:
is there a getter we can use like this.getState()?

Reply:
That's how state variables seemed to be stored. I can add a getter for that